src/components/Article/Article.tsx

```diff
@@ -121,6 +123,26 @@ const Article = ({
     return md;
   }, []);
 
+  const location = useLocation();
+
+  // Scroll to element with ID passed in as a query-parameter.
+  // We use query-params instead of the regular fragments since
+  // the article doesn't exist on initial page load (At least without SSR).
+  useEffect(() => {
+    if (location.hash && article.content) {
+      const element = document.getElementById(location.hash.slice(1));
+      const elementTop = element?.getBoundingClientRect().top ?? 0;
+      const bodyTop = document.body.getBoundingClientRect().top ?? 0;
+      const absoluteTop = elementTop - bodyTop;
+      const scrollPosition = absoluteTop - MastheadHeightPx;
```

Comment:
Om noen har noe bedre måte å håndtere dette på så er jeg all for it, denne måten fungerer, men den er ikke 100% konsekvent (Noen ganger så scroller den ikke helt ned til headeren, men jeg har ikke klart å finne ut hvorfor, men siden det skjer bare innimellom så antar jeg at det har noe med hvor langt siden har kommet i renderingen/lastingen).